src/modeler/AutoRest.Swagger.Tests/Resource/Swagger/Validation/property-names-casing.json

```diff
@@ -88,6 +88,15 @@
           "type": "string",
           "description": "Required. Gets or sets the location of the resource. This will be one of the supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.). The geo region of a resource cannot be changed once it is created, but if an identical geo region is specified on update the request will succeed."
         },
+        "@odata.etag": {
+          "x-ms-client-name": "ETag",
+          "type": "string",
+          "description": "The ETag of the synonym map."
+        },
+        "tripleCAPitalCASe": {
```

Comment:
should we call the property publicIPAddress to show a real example, and keep the description you have, which explains this case?